fix_simulation/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
+
+services:
+  quickfix:
+    container_name: truex_fix_trade_simulation
+    build:
+      context: .
+      dockerfile: Dockerfile
+    platform: linux/amd64
+    volumes:
+      - .:/app
+    environment:
+      - CC=clang
+      - CXX=clang++
+      - TRUEX_USER1_MNEMONIC=${TRUEX_USER1_MNEMONIC}
+      - TRUEX_USER1_KEY_ID=${TRUEX_USER1_KEY_ID}
+      - TRUEX_USER1_KEY_SECRET=${TRUEX_USER1_KEY_SECRET}
+      - TRUEX_USER2_MNEMONIC=${TRUEX_USER2_MNEMONIC}
+      - TRUEX_USER2_KEY_ID=${TRUEX_USER2_KEY_ID}
+      - TRUEX_USER2_KEY_SECRET=${TRUEX_USER2_KEY_SECRET}
+      - TRUEX_API_ADDRESS=http://10.10.10.11:10185
```

Comment:
Replace the internal IP with uat.truex.co

Reply:
updated in https://github.com/true-markets/tools/pull/1/commits/bc5c4b51e0f30aec79ea9038edf577a018d0bc46 . used `uat1.truex.co` as it was able to resolve. My guess is uat1 resolves to uat box 1 machine.
Also had to use the port for the rest api as the endpoint wont resolve without the port. 
e.g `curl -H "Content-Type: application/json"      -H "X-TRUEX-AUTH-TIMESTAMP: $(date +%s)"      http://uat1.truex.co:9742/api/v1/instrument` works but `curl -H "Content-Type: application/json"      -H "X-TRUEX-AUTH-TIMESTAMP: $(date +%s)"      http://uat1.truex.co:9742/api/v1/instrument` yields 
```
curl: (7) Failed to connect to uat1.truex.co port 80 after 218 ms: Couldn't connect to server
```